sklearn/cluster/_spectral.py

```diff
@@ -185,10 +185,10 @@ def spectral_clustering(affinity, *, n_clusters=8, n_components=None,
           - heat kernel of the pairwise distance matrix of the samples,
           - symmetric k-nearest neighbours connectivity matrix of the samples.
 
-    n_clusters : integer, optional
+    n_clusters : integer, default=None
```

Comment:
Use `int` instead of `integer`(Python basic types).

Reply:
ok